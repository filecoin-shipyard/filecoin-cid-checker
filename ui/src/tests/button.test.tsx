import React from 'react'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'

import { Button } from '../components/button.component'

describe('<Button /> spec', () => {
  it('renders the component', async () => {
    const { container } = render(<Button />)
    expect(container).toMatchSnapshot()
  })
})
