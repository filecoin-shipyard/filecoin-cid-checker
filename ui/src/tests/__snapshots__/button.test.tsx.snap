// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> spec renders the component 1`] = `
<div>
  <button
    class="sc-bdnylx jwFGUb"
  />
</div>
`;
